libsnowflakeclient/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ project(snowflakeclient)
 add_definitions(-DLOG_USE_COLOR)
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread -fPIC -Wall")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -O2 -pthread -fPIC -Wall")
```

Comment:
This allows Valgrind to show the source code file and line number in the results.

Reply:
We may want to remove -O2 in dev